src/Clide.IntegrationTests/Adapters/SolutionToVsAdapterSpec.cs

```diff
@@ -16,7 +16,7 @@ public SolutionToVsAdapterSpec(OpenSolution11Fixture fixture)
             adapters = GlobalServiceLocator.Instance.GetExport<IAdapterService>();
         }
 
-        [VsFact]
+        [VsFact(UIThread = true)]
```

Comment:
💡  I prefer to apply this at the assembly level, and then make all the test methods asynchronous.

Reply:
This is something we should not need too (same as the ReuseInstance usage in this PR). I need also to investigate and remove it or fix the code it's expecting this.